src/main/scala/com/databricks/labs/overwatch/pipeline/InitializerFunctionsUCE.scala

```diff
@@ -0,0 +1,210 @@
+      if (spark.catalog.databaseExists(consumerDBName)) {
+        val consumerDBMeta = spark.sessionState.catalog.getDatabaseMetadata(consumerDBName)
+
+        val existingConsumerDBLocation = spark.sql(s"describe database ${consumerDBName}")
+          .where("database_description_item ='RootLocation'")
+          .select("database_description_value").collect.map(_.getString(0)).head.toString
+
+        if (existingConsumerDBLocation != consumerDBLocation) { // separated consumer DB but same location FAIL
+          switch = false
+          throw new BadConfigException(s"The Consumer DB: $consumerDBName exists" +
+            s"at location $existingConsumerDBLocation which is different than the location entered in the config. Ensure" +
+            s"the DBName is unique and the locations match. The location must be a fully qualified URI such as " +
+            s"dbfs:/...")
+        }
+      } else { // consumer DB is different from ETL DB AND db does not exist
+        if (!Helpers.pathExists(consumerDBLocation)) { // consumer db path is empty
+          logger.log(Level.INFO, s"Consumer DB location " +
+            s"is valid: will create database: $consumerDBName at location: ${consumerDBLocation}")
+        } else {
+          switch = false
+          throw new BadConfigException(
+            s"""The consumer database location: ${dbLocation}
+          already exists. Please specify a path that doesn't yet exist. If attempting to launch Overwatch on a secondary
+          workspace, please choose a unique location for the database on this workspace.""".stripMargin)
+        }
+      }
+
+      if (consumerDBLocation == dbLocation && consumerDBName != dbName) { // separate db AND same location ERROR
+        switch = false
+        throw new BadConfigException("Consumer DB Name cannot differ from ETL DB Name while having the same location.")
+      }
+    } else { // same consumer db as etl db
+      if (consumerDBName == dbName && consumerDBLocation != dbLocation) { // same db AND DIFFERENT location ERROR
+        switch = false
+        throw new BadConfigException("Consumer DB cannot match ETL DB Name while having different locations.")
+      }
+    }
+    spark.sessionState.catalogManager.setCurrentCatalog(etlCatalogName)
+    switch
+  }
+    /**
+     * validate and set the DataTarget for Overwatch
+     * @param dataTarget OW DataTarget
+     */
+    override def validateAndSetDataTarget(dataTarget: DataTarget): Unit = {
+      // Validate data Target
+      // todo UC enablement
+
+      val etlCatalogName = dataTarget.databaseName.get.split("\\.").head
+      spark.sessionState.catalogManager.setCurrentCatalog(etlCatalogName)
+      if (!disableValidations) dataTargetIsValid(dataTarget)
+
+      // If data target is valid get db name and location and set it
+      val dbName = dataTarget.databaseName.get.split("\\.").last
+      val dbLocation = dataTarget.databaseLocation.getOrElse(s"dbfs:/user/hive/warehouse/${dbName}.db")
```

Comment:
dbLoc and consumerDBLocs are not needed for UC as customers will be creating the schemas and provisioning access; thus it doesn't matter where the databases are since all the tables are external